tests/snmpsim/ipolis.snmprec

```diff
@@ -0,0 +1,2 @@
+1.3.6.1.2.1.1.1.0|4|SAMSUNG iPolis H.264 Premium Level IP Camera
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.36849
```

Comment:
This needs to be the full oid: `.1.3.6.1.4.1.36849.1.2.88`

Reply:
Mines were .1.3.6.1.4.1.36849.1.2.91 so I wrote that